### PR TITLE
Screenshots updated fixes #8458

### LINKED ISSUE #8458: 10 Module Plugins
Update screenshots for chapter: https://docs.qgis.org/testing/en/docs/training_manual/qgis_plugins/index.html
see issue #2441

### PR DESCRIPTION
**Pull request by selmaVH1**
fixes #8458 
Screenshots updated for chapter 9. Module: Plugins

### CODE REVIEW

File: docs/training_manual/qgis_plugins/img/quickosm_setup.png
Comment (DelazJ) [inline docs/training_manual/qgis_plugins/img/quickosm_setup.png:1]: Can you resize this screenshot, please? It is unnecessarily too big.

**Review by DelazJ — COMMENTED** [review]
@selmaVH1 there are also the dataplotly screenshots to resize, they are way too big and airy (compared to the others, and for what they are showing). Thanks

File: docs/training_manual/qgis_plugins/img/quickosm_setup.png
Comment (selmaVH1) [reply docs/training_manual/qgis_plugins/img/quickosm_setup.png:1]: @DelazJ You can check it now.

**Comment by DelazJ** [comment]
> there are also the dataplotly screenshots to resize, they are way too big and airy

Sorry Selma, i wasn't clear enough. I was talking about the screenshots at https://docs.qgis.org/testing/en/docs/training_manual/qgis_plugins/plugin_examples.html#basic-fa-the-dataplotly-plugin that were made too high.

**Comment by DelazJ** [comment]
> this DataPlotly plugin doesn't work for me

Confirmed. It requires pandas installation. This is not mentioned in the install or metadata details (and actually the opposite is mentioned at https://github.com/ghtmtt/DataPlotly/tree/master#usage). Needs an update, @ghtmtt? 
> Is it ok to resize existing images or should I take new ones?

While at checking and fixing the install error, I updated the screenshots in #8476

**Pull request merged**